src/features/Content/MoviesList/Tile/index.js

```diff
@@ -9,19 +9,20 @@ const Tile = ({ movieBrowser }) => {
             {movieBrowser.map(({ id, poster_path, original_title, release_date, genre_ids, vote_average, vote_count }) => (
                 <Container key={id}>
                     <Image src={img_base_url + poster_path} alt={original_title} />
-                    <Description>
-                        <Title> {original_title} </Title>
-                        {/* <Year>{release_date.substring(4,-1)}</Year> */}
-                        <Tags>
-                            <Tag>{genre_ids}</Tag>
-                        </Tags>
-
+                    <Information>
+                        <Description>
+                            <Title> {original_title} </Title>
+                            {/* <Year>{release_date.substring(4, -1)}</Year> */}
```

Comment:
trzeba zrobić coś z tym year ale wiem że to w przyszłości ale przypominajka taka